cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/MetricsHandlers.java

```diff
@@ -0,0 +1,213 @@
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+ */
+package com.telefonica.iot.cygnus.management;
+
+import com.google.common.collect.ImmutableMap;
+import com.telefonica.iot.cygnus.handlers.CygnusHandler;
+import com.telefonica.iot.cygnus.metrics.CygnusMetrics;
+import com.telefonica.iot.cygnus.log.CygnusLogger;
+import com.telefonica.iot.cygnus.sinks.CygnusSink;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.flume.Sink;
+import org.apache.flume.SinkProcessor;
+import org.apache.flume.SinkRunner;
+import org.apache.flume.Source;
+import org.apache.flume.SourceRunner;
+import org.apache.flume.source.http.HTTPSourceHandler;
+
+/**
+ *
+ * @author frb
+ */
+public final class MetricsHandlers {
+    
+    private static final CygnusLogger LOGGER = new CygnusLogger(MetricsHandlers.class);
+    
+    /**
+     * Constructor. Utility classes should not have a public or default constructor.
+     */
+    private MetricsHandlers() {
+    } // MetricsHandlers
+    
+    /**
+     * Handles GET /v1/admin/mergeMetrics and /admin/mergeMetrics.
+     * @param request
+     * @param response
+     * @param sources
+     * @param sinks
+     * @throws IOException
+     */
+    public static synchronized void get(HttpServletRequest request, HttpServletResponse response,
+            ImmutableMap<String, SourceRunner> sources, ImmutableMap<String, SinkRunner> sinks) throws IOException {
+        // Let's assume everything goes well
+        response.setStatus(HttpServletResponse.SC_OK);
+        
+        // Get the reset parameter
+        String reset = request.getParameter("reset");
+        
+        if (reset != null && !reset.equals("true") && !reset.equals("false")) {
+            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+            return;
+        } // if
+        
+        if (reset != null && reset.equals("true")) {
+            CygnusMetrics metrics = mergeMetrics(sources, sinks);
+            response.getWriter().println(metrics.toJsonString());
+            deleteMetrics(sources, sinks);
+            response.setContentType("application/json; charset=utf-8");
+        } else {
+            CygnusMetrics metrics = mergeMetrics(sources, sinks);
+            response.getWriter().println(metrics.toJsonString());
+            response.setContentType("application/json; charset=utf-8");
+        } // else
+    } // getServiceSubserviceMetrics
+    
+    /**
+     * Handles DELETE /v1/admin/mergeMetrics and /admin/mergeMetrics.
```

Comment:
Same here.

Maybe they are typos in the comments and they actually refer to `/admin/metrics` ?